src/org/rascalmpl/library/Location.rsc

```diff
@@ -43,6 +44,18 @@ loc relativize(list[loc] haystack, loc needle) {
     }
 }
 
+@synopsis{Convert Windows path syntax to a `loc` value}
+@description{
+This conversion supports generic Windows path syntax, including:
+* Absolute drive-specific: `C:\Program Files`
+* Relative drive-specific: `C:hello.txt`
+* Relative: `hello.txt`
+* Directory-relative: `\hello.txt`
+* UNC format: `\\\\system07\\C$\\`
```

Comment:
the other 2 lines (53 & 50) don't escape the `\`?

Reply:
check! copy/paste issue. I'd like the literal value in the docs even though the string version has to be escaped.